### DISCUSSION TITLE
Add / remove song to/from a playlist fails

### DISCUSSION THREAD

**Discussion by one-step-behind** [rern/rAudio#1537] [Bug Reports]
I was waiting a long time to upgrade rAudio to the latest version (20231216) until most of the issues reported by users were solved by @rern. Now I did it and faced an issue when adding a song to a playlist and removing one. Found the root cause:

tl;dr:
`#pl-path .lipath` => `#savedpl-path .lipath`

Long version:

`var plname = $( '#pl-path .lipath' ).text();` has to be changed to `var plname = $( '#savedpl-path .lipath' ).text();` in:

1. `/srv/http/assets/js/function.js#973`
2. `/srv/http/assets/js/context.js#255`

`/srv/http/assets/js/passive.js` has probably to be fixed, too (dunno what these functions are doing):
(at least these placed were found by a `find /srv/http/ -type f -exec grep -l "#pl-path .lipath" {} \;`)

1. `/srv/http/assets/js/passive.js#297`:
`var name = $( '#pl-path .lipath' ).text();` => `var name = $( '#savedpl-path .lipath' ).text();`
2. `/srv/http/assets/js/passive.js#391`:
`if ( 'delete' in data && $( '#pl-path .lipath' ).text() === data.delete ) $( '#playlist' ).trigger( 'click' );` => `if ( 'delete' in data && $( '#savedpl-path .lipath' ).text() === data.delete ) $( '#playlist' ).trigger( 'click' );`

So, in short the selector to get the playlist name doesn't match and will be empty in both cases (add & remove) which leads into a playlist file called "`.m3u`" (without a name).

Would be nice if this could be sorted out. Are pull requests welcome?
Thank you @rern for all the work and time you put into this project.

Merry Christmas! 🎄🎅

**Comment by one-step-behind** [comment]
Seems to be fixed by rAudio 20240108.
